--- autoconf-archive.rb	2012-12-04 22:32:38.000000000 -0500
+++ Portfile	2012-09-26 13:00:20.000000000 -0400
@@ -1,13 +1,23 @@
-require 'formula'
+# $Id: Portfile 98158 2012-09-26 16:42:08Z adfernandes@macports.org $
 
-class AutoconfArchive < Formula
-  homepage 'http://savannah.gnu.org/projects/autoconf-archive/'
-  url 'http://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2012.11.14.tar.bz2'
-  mirror 'http://ftp.gnu.org/gnu/autoconf-archive/autoconf-archive-2012.11.14.tar.bz2'
-  sha1 'b2bcc46fb9b5ad3da0a3cb4014ed2e0d2b52acec'
+PortSystem 1.0
 
-  def install
-    system './configure', "--prefix=#{prefix}"
-    system 'make install'
-  end
-end
+name                autoconf-archive
+version             2012.09.08
+license             Permissive
+categories          devel
+maintainers         adfernandes
+description         A collection of over 500 reusable Autoconf macros
+long_description    The Autoconf Macro Archive aims to provide a collection of reusable Autoconf \
+    macros as free software. The archive currently features more than 500 macros \
+    which perform portability tests ranging from compiler support for weird language \
+    extensions to automatic generation of sophisticated Automake rules. All these \
+    macros have been contributed by friendly supporters of the cause from all over \
+    the Internet - this archive is merely a distribution of other people's efforts.
+homepage        http://www.gnu.org/software/autoconf-archive/
+master_sites    gnu
+use_bzip2       yes
+checksums       rmd160  2231e79be018446d45483065bd6a0a2e7fc805c7 \
+                sha256  bc5dab3edd4fd337f8f9b71dd44a0382d15a28df5174c0632b9cee0449e2affc
+
+platforms       darwin
